### PR TITLE
B2c exports 2.0 updates

### PR DESCRIPTION
**Pull request by abhinavv245**

### CODE REVIEW

**Review by tanya-ondc — CHANGES_REQUESTED** [review]

**Comment by abhinavv245** [comment]
Updated the PR

File: api/components/Examples/B2C_Exports/init/init_exports.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                city:
+                  name: Jurong East
+                country:
+                  code: SGP
+                area_code: "680230"
+                state:
+                  name: ""
+              contact:
+                phone: "9886098860"
+          customer:
```
Comment (tanyamadaan) [inline api/components/Examples/B2C_Exports/init/init_exports.yaml:62]: had asked to check and update this in select as well?

File: api/components/Examples/B2C_Exports/on_cancel/on_cancel.yaml [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+            currency: USD
+            transaction_id: "1567"
+            amount: "15.00"
+          status: PAID
+          "@ondc/org/buyer_app_finder_fee_type": percent
+          "@ondc/org/buyer_app_finder_fee_amount": "0"
+          "@ondc/org/settlement_basis": delivery
+          "@ondc/org/settlement_window": P1D
+          "@ondc/org/withholding_amount": "10.00"
+          "@ondc/org/settlement_details":
```
Comment (tanyamadaan) [inline api/components/Examples/B2C_Exports/on_cancel/on_cancel.yaml:247]: do we need both P2 as well this settlement details for refund?

File: api/components/Examples/B2C_Exports/on_cancel/on_cancel_merchant.yaml [hunk 3]
```diff
@@ -113,111 +101,110 @@ value:
                   end: '2023-02-03T11:30:00.000Z'
                 timestamp: '2023-02-03T11:35:00.000Z'
```
Comment (tanyamadaan) [inline api/components/Examples/B2C_Exports/on_cancel/on_cancel_merchant.yaml:102]: how come it is delivered ? and then being cancelled? remove timestamp

File: api/components/Examples/B2C_Exports/on_cancel/on_cancel_merchant.yaml [hunk 4]
```diff
@@ -113,111 +101,110 @@ value:
...
               instructions:
-                name: Proof of delivery
-                short_desc: Proof of delivery details
-                long_desc: Proof of delivery details
-                images:
-                  - https://image1_url.png
+                name: Status for delivery
```
Comment (tanyamadaan) [inline api/components/Examples/B2C_Exports/on_cancel/on_cancel_merchant.yaml:104]: This as well, as product is not yet delivered.

File: api/components/Examples/B2C_Exports/on_cancel/on_cancel_merchant.yaml [hunk 5]
```diff
@@ -227,47 +214,49 @@ value:
...
+            currency: USD
+            transaction_id: "1567"
+            amount: "15.00"
           status: PAID
           "@ondc/org/buyer_app_finder_fee_type": percent
           "@ondc/org/buyer_app_finder_fee_amount": "0"
           "@ondc/org/settlement_basis": delivery
           "@ondc/org/settlement_window": P1D
           "@ondc/org/withholding_amount": "10.00"
+          "@ondc/org/settlement_details":
```
Comment (tanyamadaan) [inline api/components/Examples/B2C_Exports/on_cancel/on_cancel_merchant.yaml:251]: ?

**Review by tanyamadaan — CHANGES_REQUESTED** [review]

**Review by tanyamadaan — APPROVED** [review]

**Pull request merged**